spec/text-editor-spec.coffee

```diff
@@ -2899,6 +2899,35 @@ describe "TextEditor", ->
               expect(buffer.lineForRow(3)).toBe '    var pivot = item'
               expect(atom.clipboard.read()).toBe ' <= 1) ret\ns.shift(), current, left = [], right = [];'
 
+      describe ".cutToEndOfBufferLine()", ->
+        describe "when soft wrap is on", ->
+          it "cuts up to the end of the buffer line", ->
+            editor.setSoftWrapped(true)
+            editor.setEditorWidthInChars(10)
+            editor.setCursorScreenPosition([2, 2])
+            editor.cutToEndOfBufferLine()
+            expect(editor.tokenizedLineForScreenRow(2).text).toBe '= '
+
+        describe "when soft wrap is off", ->
```

Comment:
> I might eliminate the soft wrap on / off sections and just test everything with soft wrap on.

:sparkles: I agree -- I removed that in bed902e and now testing everything with soft wrap on.